### PR TITLE
Readme: Order more appropriately

### PR DESCRIPTION
**Pull request by xeruf**
- little wording adjustments
- reorder sections to fit user interest

### CODE REVIEW

**Comment by aaronliu0130** [comment]
closed due to 380cd0a

File: README.md [hunk 1]
```diff
@@ -1,48 +1,54 @@
...
+
+Debian/Ubuntu/RPM:
+```sh
+sudo apt install imagemagick || yum install ImageMagick
+git clone https://github.com/stefanhaustein/TerminalImageViewer.git
+cd TerminalImageViewer/src/main/cpp
+make
+sudo make install
+```
+Arch: Install `terminalimageviewer-git` from the AUR
```
[the remark this answers is not in the document]
Comment (xeruf) [reply README.md:37]: https://aur.archlinux.org/packages/terminalimageviewer-git
On the right, "View PKGBUILD"

[the remark this answers is not in the document]
Comment (aaronliu0130) [reply README.md:37]: > <html>
> <body>
> 
> Last Updated: | 2018-04-02 13:51 (UTC)
> -- | --
> 
> </body>
> </html>
Yep, outdated.

[the remark this answers is not in the document]
Comment (xeruf) [reply README.md:37]: this is built from the latest git version, thus the buildscript does not have to be revised on new versions...

[the remark this answers is not in the document]
Comment (aaronliu0130) [reply README.md:37]: the arch, pkgver and three names still have to be revised.

**Pull request closed without merging**